CMakeLists.txt

```diff
@@ -39,6 +39,9 @@ option(INSTALL_COMPACT_FILES "Copy compact files to install area" OFF)
 
 
 find_package(DD4hep REQUIRED COMPONENTS DDRec DDG4 DDParsers)
+# dd4hep_set_compiler_flags() expects DD4hep_SET_RPATH to be set to ON
```

Comment:
It depends what you mean by "this", because when xerces was in `LD_LIBRARY_PATH` then it didn't matter whether the RPATH was being set or not since the libraries will be found anyways. If by "this" you mean the function `dd4hep_set_compiler_flags` then I have no idea if this ever set up the RPATH for a package that is not DD4hep

Reply:
Ah sorry. *this* was the fact, that I thought dd4hep set up `RPATH` properly with `dd4hep_set_compiler_flags` (without having to set `DD4hep_SET_RPATH` before that)

Reply:
This is what is run without that variable so it's very much needed: https://github.com/AIDASoft/DD4hep/blob/master/cmake/DD4hepBuild.cmake#L135. It seems it was introduced here: https://github.com/AIDASoft/DD4hep/pull/654 but the previous line was there already

Reply:
OK. Then I was misremembering. Thanks for the clarification.